### ISSUE TITLE
Error: connect EHOSTDOWN 169.254.169.254:80 - Local

### ISSUE THREAD

**Issue by PythonicNinja** [syndbg/webpack-google-cloud-storage-plugin#8]
@syndbg @bw @youanden @chown9835 
`Error: connect EHOSTDOWN 169.254.169.254:80 - Local`

my webpack.config.js:
```
new WebpackGoogleCloudStoragePlugin({
      directory: './dist',
      // NOTE: Array of filenames to include in the uploading process
      include: ['.js', '.html', '.css', '.svg', '.png', '.jpg'],
      // NOTE: Array of filenames to exclude in the uploading process
      // exclude: ['cats.js'],
      // NOTE: Options passed directly to
      // Google cloud Node Storage client.
      // This is mostly authentication-wise.
      // For more information:
      // https://github.com/GoogleCloudPlatform/google-cloud-node/tree/master/packages/storage#authentication
      storageOptions: {
        projectId: 'project_id',
      },
      // NOTE: Options used by
      // WebpackGoogleCloudStoragePlugin
      // regarding uploading.
      uploadOptions: {
        // Where to upload files
        bucketName: 'bucket-with-my-domain.com',
        destinationNameFn: file => file.path.replace('dist/', ''),
        forceCreateBucket: true,
        gzip: true,
        makePublic: true,
      },
    }),
```

traceback:
```
events.js:163
      throw er; // Unhandled 'error' event
      ^

Error: connect EHOSTDOWN 169.254.169.254:80 - Local (192.168.1.143:51072)
    at Object.exports._errnoException (util.js:1050:11)
    at exports._exceptionWithHostPort (util.js:1073:20)
    at internalConnect (net.js:889:16)
    at lookupAndConnect (net.js:977:5)
    at Socket.realConnect (net.js:945:5)
    at Agent.connect [as createConnection] (net.js:77:22)
    at Agent.createSocket (_http_agent.js:195:26)
    at Agent.addRequest (_http_agent.js:157:10)
    at new ClientRequest (_http_client.js:212:16)
    at Object.request (http.js:26:10)
    at Request.start (/node_modules/google-auto-auth/node_modules/request/request.js:740:32)
    at Request.end (/node_modules/google-auto-auth/node_modules/request/request.js:1394:10)
    at end (/node_modules/google-auto-auth/node_modules/request/request.js:567:14)
    at Immediate.<anonymous> (/node_modules/google-auto-auth/node_modules/request/request.js:581:7)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

**Comment by PythonicNinja** [comment]
Changed to `"@google-cloud/storage": "1.5.2"`

**Comment by PythonicNinja** [comment]
Will do PR with that.

**Comment by syndbg** [comment]
Can we consider this fixed after the PR was merged and `0.8.0` was released?

**Comment by PythonicNinja** [comment]
Sure! Thanks!
